miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/rest/ContainerRestController.java

```diff
@@ -102,6 +102,7 @@ protected ContainerDto asDto(SequencerPartitionContainer model) {
     if (platformType == null) {
       throw new RestException("Invalid platform.", Status.BAD_REQUEST);
     }
-    return jQueryBackend.get(request, response, uriBuilder, PaginationFilter.platformType(platformType));
+    DataTablesResponseDto<ContainerDto> dto = jQueryBackend.get(request, response, uriBuilder, PaginationFilter.platformType(platformType));
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "dto". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1488)